src/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -35,12 +36,25 @@ public static void Ctor_ByteArray(byte[] b, Guid expected)
             Assert.Equal(expected, new Guid(b));
         }
 
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public static void Ctor_ReadOnlySpan(byte[] b, Guid expected)
+        {
+            Assert.Equal(expected, new Guid(new ReadOnlySpan<byte>(b)));
```

Comment:
Tests of new APIs that are netcoreapp specific will need to be done in a file that's only built into the netcoreapp build... otherwise, for example, the tests will fail to compile against netfx.